ui/lib/pki/addon/routes/configuration/tidy.js

```diff
@@ -18,7 +18,7 @@ export default class PkiConfigurationTidyRoute extends Route {
 
     return hash({
       engine: this.modelFor('application'),
-      tidy: this.store.createRecord('pki/tidy', { backend }),
+      tidy: this.store.createRecord('pki/tidy', { backend, type: 'manual-tidy' }),
```

Comment:
🤔 Hmm, so adding this here makes it an attribute type which might make things confusing in the model. It's nice to, when we can, keep the model attributes aligning with the API parameters as much as possible. What about instead passing it in to the `adapterOptions` during the action that calls the save() method?

Reply:
This makes sense! I didn't know you can do it on `save()`! Pretty cool!